src/edn/error.rs

```diff
@@ -0,0 +1,91 @@
+use core::fmt::{self, Debug, Display};
+
+pub struct Error {
+    code: Code,
+    // line: usize,   // TODO walker
+    // column: usize, // TODO walker
+}
+
+#[non_exhaustive]
+pub enum Code {
+    /// Catchall/placeholder error messages
+    Message(Box<str>),
+    Io(std::io::Error),
+    TryFromInt(std::num::TryFromIntError),
+    #[doc(hidden)]
+    Infallable(), // Makes the compiler happy for converting u64 to u64 and i64 to i64
+}
+
+impl Error {
```

Comment:
I think this functions should be snake_cased

Reply:
These functions aren't intended to remain in the code base. It's only a super hacky way to make things 
like `return Err(Error::ParseEdn(...))` compile without changing the code. 

I put TODOs in the comments, but a walker wouldn't be using these at all.